test/omnisharp/omnisharpUnitTests/logging/errorMessageObserver.test.ts

```diff
@@ -16,7 +15,7 @@ describe('ErrorMessageObserver', () => {
     let errorMessage: string | undefined;
     const observer = new ErrorMessageObserver(vscode);
 
-    vscode.window.showErrorMessage = async (message: string, ..._: string[]) => {
+    vscode.window.showErrorMessage = async <T extends MessageItem>(message: string, ..._items: T[]) => {
         errorMessage = message;
         return Promise.resolve<string>('Done');
```

Comment:
This seems suspect now that we've changed a lot of places to use ActionItems and such.

Reply:
this is mocking the VSCode API call, which uses `MessageItem` objects.  The actual `errorMessageObserver.ts` will call our helper with `ActionOption` (or w.e other one) which then calls the VSCode API (which we mock here).